R/html_document_base.R

```diff
@@ -65,6 +66,23 @@ html_document_base <- function(theme = NULL,
     if (is_bs_theme(theme)) bslib::bs_global_set(old_theme)
   }
 
+  # additional sass/css
+  for (f in css) {
+    if (is_bs_theme(theme)) {
+      theme <- bslib::bs_add_rules(theme, sass::sass_file(f))
+      next
+    }
+    is_sass <- grepl("\\.s[ac]ss$", f)
+    if (is_sass) {
+      f <- sass::sass(
```

Comment:
We missed that in previous PR but should we check for sass availability and throw a nice error early if **sass** is not installed yet ? 
```suggestion
      if (!xfun::loadable("sass")) {
        stop("Using `.sass` or `.scss` file in `css` argument requires the sass package.", call. = FALSE)
      }
      f <- sass::sass(
```